cluster-service/config.tmpl.mk

```diff
@@ -0,0 +1,13 @@
+REGION ?= {{ .region }}
+CONSUMER_NAME ?= {{ .maestroConsumerName }}
+RESOURCEGROUP ?= {{ .serviceClusterRG }}
+REGIONAL_RESOURCEGROUP ?= {{ .regionRG }}
+AKS_NAME ?= {{ .aksName }}
+SERVICE_KV ?= {{ .serviceKeyVaultName }}
+OIDC_STORAGE_ACCOUNT ?= {{ .oidcStorageAccountName }}
+IMAGE_REPO ?= {{ .clusterServiceImageRepo }}
+IMAGE_TAG ?= {{ .clusterServiceImageTag }}
+ACR_NAME ?= {{ .acrName }}
+AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID ?= {{ .firstPartyAppClientId }}
+FPA_CERT_NAME ?= firstPartyCert
+ZONE_RESOURCE_ID ?= $(shell az network dns zone show -n {{ .regionalDNSSubdomain }}.{{ .baseDnsZoneName }} -g {{ .regionRG }} --query id -o tsv)
```

Comment:
How do you choose to put imperative calls here versus in the `Make` targets themselves?

Reply:
if a variable is used by more than one target, it is placed outside, otherwise inside
need to check if that really holds true.
let me know if you have suggestions around that